pages/seller.vue

```diff
@@ -28,15 +46,34 @@ import Mycatogory from "~/components/Category";
 import Productcard from "~/components/GoodsCard";
 import Mainitems from "~/assets/data/main-items";
 import Pagination from "~/components/Pagination";
+import DeleteAddModal from "../components/sellerPage/deleteAddModal";
+
 export default {
-  components: { Mysuggestion, Mycatogory, Productcard, Pagination },
+  components: {
+    DeleteAddModal,
+    Mysuggestion,
+    Mycatogory,
+    Productcard,
+    Pagination,
+  },
   name: "seller",
   data() {
     return {
       mainitems: Mainitems,
+      deleteModalIsVisible: false,
     };
   },
+  computed: {},
+  methods: {
+    showDeleteModal(val) {
+      this.deleteModalIsVisible = true;
+      console.log(val.name);
```

Comment:
Do not leave log in, it's good practice keep console clean of logs and use logs only while development but do not commit it.